rasa/core/training/interactive.py

```diff
@@ -1603,17 +1615,19 @@ def start_visualization(image_path: Text, port: int) -> None:
 
     # noinspection PyUnusedLocal
     @app.exception(NotFound)
-    async def ignore_404s(request, exception):
+    async def ignore_404s(request: Any, exception: Any) -> HTTPResponse:
         return response.text("Not found", status=404)
 
     # noinspection PyUnusedLocal
     @app.route(VISUALIZATION_TEMPLATE_PATH, methods=["GET"])
-    def visualisation_html(request):
+    def visualisation_html(request: Any) -> Coroutine[Any, Any, HTTPResponse]:
```

Comment:
why `Coroutine`?
```suggestion
    def visualisation_html(request: Request) -> HTTPResponse:
```

Reply:
I was guiding myself by PyCharm suggestions looking at what this function returns (IDE warns `Expected type 'HTTPResponse' , got 'Coroutine' instead` )

Reply:
seems Pycharm can't really deal with coroutines and type annoations yet 🤔